src/Management/Nomadic.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Thinktomorrow\Chief\Management;
+
+use Thinktomorrow\Chief\Management\Exceptions\NotAllowedManagerRoute;
+
+trait Nomadic
+{
+    public function nomadicCan($verb)
+    {
+        if ($this->indexCollection()->count() > 0 && in_array($verb, ['create', 'store'])) {
+            throw NotAllowedManagerRoute::create($this);
+        }
+
+        if (! auth()->guard('chief')->user()->hasRole('developer') && in_array($verb, ['create', 'store', 'delete'])) {
```

Comment:
kleintje maar mss dit tweede if statement vooraan zetten. Op die manier wordt niet altijd the indexCollection opgehaald wat toch een expensive operation is. 

in die andere lijn kan voor dezelfde optimalisatie de in_array check vooraan in statement worden gezet